ci-operator/step-registry/openshift/e2e/aws/loki/OWNERS

```diff
@@ -1,2 +1,4 @@
 approvers:
 - ingvagabund
```

Comment:
And similar changed to `openshift/e2e/gcp/loki`?  I expect `openshift/upgrade/gcp/loki` should be @vrutkovs and me and any other interested parties, but I can handle that in follow-up work after this lands.